### DISCUSSION TITLE
HomematicIP Geräte lassen sich aus HA nur einmal schalten.

### DISCUSSION THREAD

**Discussion by zshermann** [SukramJ/aiohomematic#1202] [Q&A]
Hallo,

Geräte welche über HA Integration Homematic(IP) Local eingefügt sind, lassen sich genau einmal schalten und danach nicht mehr.
Eine Steckdose z.B. lässt sich einaschalten aber nicht ausschalten.

System(e):

HA: auf einem Raspi 4 - 4GB RAM, 128 SSD per USB 3
Home Assistant 2023.9.3
Supervisor 2023.09.2
Operating System 10.5
Frontend 20230911.0 - latest

Raspmatic: Raspi3 3 GB, 4GB Ram 32GB SDCard
                   Raspmatic Version: 3.71.12.20230826 (rpi3)

Habe meine HA Installation auch schon mal komplett neu hochgezogen, aber null Erfolg.
Hat da jemand eine Idee?

Gruß ZsHermann

**Comment by danielperna84** [comment]
Ohne Logs kann man leider nicht helfen. ☝️

**Comment by zshermann** [comment]
Habe da mal ein gestern ein log erstellt.
Ich hoffe es hilft.

[home-assistant_homematicip_local_2023-09-25T16-45-56.201Z.log](https://github.com/danielperna84/hahomematic/files/12724926/home-assistant_homematicip_local_2023-09-25T16-45-56.201Z.log)

**Reply by SukramJ** [reply]
Da du zusätzlich ein Issue erstellt hast werde ich diese Diskussion schliessen.
